scripts/lib/sphinx/generateToc.ts

```diff
@@ -125,10 +126,15 @@ export function generateToc(options: {
     tocChildren.push(...orderEntriesByTitle(nestedTocModules));
   }
 
-  tocChildren.push({
-    title: pkg.name === "qiskit" ? "Changelog" : "Release notes",
-    url: pkg.releaseNotesUrl,
-  });
+  const releaseNoteEntry: TocEntry = {
+    title: "Release notes",
+  };
+  if (pkg.releaseNoteEntries) {
```

Comment:
Wait is this right? I'd think this should be negated.

Reply:
Oops yes that's wrong. It was working because empty lists are truthy in js and the tests currently only test packages with single-file release notes. Fixed in e14ae29cf149b3764ab16c92c3dcbda6b9b5ca40 and I'll add a new test in #352.